src/components/common/header/PageHeader.tsx

```diff
@@ -1,47 +1,53 @@
 import React from 'react'
 import Tippy from '@tippyjs/react'
+import { ReactComponent as Close } from '../../../assets/icons/ic-close.svg'
+
 export interface PageHeaderType {
     headerName?: string
-    buttonText?: string
     onClickCreateButton?: () => void
```

Comment:
can we remove onClickCreateButton,  showCreateButton?

Reply:
Yes done